src/OrchardCore.Modules/OrchardCore.Contents/Services/DefaultContentsAdminListFilterProvider.cs

```diff
@@ -144,26 +139,21 @@ public void Build(QueryEngineBuilder<ContentItem> builder)
                             {
                                 // We display a specific type even if it's not listable so that admin pages
                                 // can reuse the Content list page for specific types.
-                                var contentItem = await contentManager.NewAsync(contentTypeDefinition.Name);
-                                contentItem.Owner = userNameIdentifier;
-                                var listPermissionToType = ContentTypePermissionsHelper.CreateDynamicPermission(ContentTypePermissionsHelper.PermissionTemplates[CommonPermissions.ListContent.Name], contentTypeDefinition);
-                                var hasContentListPermission = await authorizationService.AuthorizeAsync(user, listPermissionToType, contentItem);
-                                if (hasContentListPermission)
-                                {
-                                    query.With<ContentItemIndex>(x => x.ContentType == contentType);
-                                }
-                                else
+
+                                // It is important to pass null to the owner parameter. This will check if the user can edit or view content that belongs to others.
+                                if (await authorizationService.AuthorizeContentTypeAsync(user, CommonPermissions.EditContent, contentTypeDefinition.Name, owner: null)
+                                || await authorizationService.AuthorizeContentTypeAsync(user, CommonPermissions.ViewContent, contentTypeDefinition.Name, owner: null))
```

Comment:
check if we can only use ViewContent in that case. We usually use EditContent to prevent showing the buttons.

Reply:
@sebastienros done!